src/app/zap-templates/zcl/zcl.json

```diff
@@ -771,7 +771,11 @@
             "FeatureMap"
         ],
         "Camera AV Settings User Level Management": ["MPTZPosition"],
-        "Soil Measurement": ["SoilMoistureMeasurementLimits"],
+        "Soil Measurement": [
+            "SoilMoistureMeasurementLimits",
```

Comment:
I was told from people who know better that Structs and lists are automatically AAI assigned and don't need to be listed here and the -extensions.json as well.

Reply:
Oh ok, I mostly put it there since it then forces the ZAP UI for force it to external, but if we do not need it I will just remove it.